.travis.yml

```diff
@@ -106,61 +61,16 @@ jobs:
               - docker exec -it travis_coverity_scan coverity.sh build
               - docker exec -it travis_coverity_scan coverity.sh upload
 
-        - stage: clean docker
-          language: python
-          # python:
-              # - "3.6"  Probably broken ATM
-          env:
-              - SIZE_LIMIT="3000" # Limit in MBs
-              - TAG_LIMIT="3" # Number of tags to be kept at the time
-          before-script:
-              - sudo apt-get -y install python3
-          script:
-                # Get docker-remote tool and setup venv
-              - sudo $CI_TOOL_DIR/get-docker-remote.sh
-                # Activate virtual environment to be able to use docker-remote safely
-              - source venv/bin/activate
-              # Check the size and tag limit of the repo
-              - REPO_SIZE=$(docker-remote repository --size $DOCKER_REPOSITORY)
-              - TAG_COUNT=$(docker-remote tags --count $DOCKER_REPOSITORY)
-              - 'echo -e "\033[33;1mCurrent repository size: $REPO_SIZE in $TAG_COUNT tags \033[0m"'
-              - |
-                  if [[ ${REPO_SIZE%.*} -gt $SIZE_LIMIT ]] || [[ $TAG_COUNT -gt $TAG_LIMIT ]]
-                  then
-                      docker-remote --login $DOCKER_USERNAME:$DOCKER_PASSWORD \
-                      tags --assumeyes --pop-back --keep $TAG_LIMIT $DOCKER_REPOSITORY
-                  fi
-
-
-
 # Specify the order of stages and conditions
 stages:
       # Helper stage to determine whether coverity stage should be allowed
     - name: initialization
```

Comment:
If I am not missing anything, I don't think you need this stage (actually, it might have been just a spillover from the time there weren't cron jobs).

Reply:
I was not entirely sure what is the purpose of that stage so I didn't touch it :). Thank you for the review!